.github/workflows/on-prerelease.yml

```diff
@@ -0,0 +1,78 @@
+# SPDX-License-Identifier: Apache-2.0
+# Licensed to the Ed-Fi Alliance under one or more agreements.
+# The Ed-Fi Alliance licenses this file to you under the Apache License, Version 2.0.
+# See the LICENSE and NOTICES files in the project root for more information.
+
+name: On Pre-Release
+on:
+  workflow_dispatch:
+  # release:
+  #   types:
+  #     - prereleased
+
+env:
+  ARTIFACTS_API_KEY: ${{ secrets.AZURE_ARTIFACTS_PERSONAL_ACCESS_TOKEN }} # | base64 # ToDo: For some reason the pipe is not returning the expected value. Do not know why yet.
+  # GITHUB_TOKEN: ${{ secrets.PAT_ATTACH_TO_RELEASE }}
+
+jobs:
+  azure-publish:
+    name: Publish Meadowlark mongodb
+    runs-on: ubuntu-latest
+    outputs:
+      hash-code: ${{ steps.hash-code.outputs.hash-code }}
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@755da8c3cf115ac066823e79a1e1788f8940201b # v3.2.0
+
+      - name: Create user .npmrc file for mongodb.
+        run: |
+          # Create the user .npmrc file with proper settings
+          touch $HOME/.npmrc
+          echo "; begin auth token" >> .npmrc
+          echo //pkgs.dev.azure.com/ed-fi-alliance/Ed-Fi-Alliance-OSS/_packaging/EdFi/npm/registry/:username=ed-fi-alliance >> $HOME/.npmrc
+          echo //pkgs.dev.azure.com/ed-fi-alliance/Ed-Fi-Alliance-OSS/_packaging/EdFi/npm/registry/:_password=${{ env.ARTIFACTS_API_KEY }} >> $HOME/.npmrc
+          echo //pkgs.dev.azure.com/ed-fi-alliance/Ed-Fi-Alliance-OSS/_packaging/EdFi/npm/registry/:email=not-used@example.com >> $HOME/.npmrc
+          echo //pkgs.dev.azure.com/ed-fi-alliance/Ed-Fi-Alliance-OSS/_packaging/EdFi/npm/:username=ed-fi-alliance >> $HOME/.npmrc
+          echo //pkgs.dev.azure.com/ed-fi-alliance/Ed-Fi-Alliance-OSS/_packaging/EdFi/npm/:_password=${{ env.ARTIFACTS_API_KEY }} >> $HOME/.npmrc
+          echo //pkgs.dev.azure.com/ed-fi-alliance/Ed-Fi-Alliance-OSS/_packaging/EdFi/npm/:email=not-used@example.com >> $HOME/.npmrc
+          echo "; end auth token" >> $HOME/.npmrc
+
+      - name: Setup Node
+        uses: actions/setup-node@8c91899e586c5b171469028077307d293428b516 # v3.5.1
+        with:
+          node-version: "16"
+          cache: "npm"
+          cache-dependency-path: "**/package-lock.json"
+
+      - name: Install dependencies
+        working-directory: Meadowlark-js
+        if: ${{ steps.modules-cache.outputs.cache-hit != 'true' }}
+        run: npm install
+
+      - name: Publish-mongodb-backend
+        working-directory: Meadowlark-js/backends/meadowlark-mongodb-backend
+        run: npm publish
```

Comment:
In an upcoming iteration, when you are ready to publish more than just mongodb, you can stay in the `Meadowlark-Js` directory and use `lerna` to publish all of the packages. Take a look at the lerna commands in MetaEd's [reusable-publish.yml](https://github.com/Ed-Fi-Closed/MetaEd-js/blob/main/.github/workflows/reusable-publish.yml).